cpp/tests/binaryop/assert-binops.h

```diff
@@ -81,7 +81,7 @@ void ASSERT_BINOP(cudf::column_view const& out,
                   TypeOp&& op,
                   ValueComparator const& value_comparator = ValueComparator())
 {
-  auto lhs_h    = static_cast<ScalarType const&>(lhs).operator TypeLhs();
+  auto lhs_h    = static_cast<ScalarType const&>(lhs).get_value(cudf::get_default_stream());
```

Comment:
Please refactor the tests like this:
```suggestion
  auto lhs_h    = static_cast<ScalarType const&>(lhs).value(cudf::get_default_stream());
```